control/controlclient/hostinfo_linux.go

```diff
@@ -121,3 +127,22 @@ func inKnative() bool {
 	}
 	return false
 }
+
+func inAwsLambda() bool {
```

Comment:
Go style says this should be `inAWSLambda` (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
Done.